.bazelignore

```diff
@@ -1,6 +1,7 @@
 # Some examples have their own WORKSPACE. Ignore as part of this root WORKSPACE so
 # we don't break trying to build separate workspaces using wildcards like //...
 # examples/dagger doesn't have its own workspace, so don't do all of examples.
-examples/android
-examples/node
-examples/trivial
+examples / android
```

Comment:
Did your linter apply starlark linting to a path?

Reply:
No -- I think it's intellij.

Reply:
Nope. Bad githook grep.